flink-ai-flow/ai_flow/client/__init__.py

```diff
@@ -212,6 +213,40 @@ def delete_project_by_name(project_name) -> Status:
     return get_ai_flow_client().delete_project_by_name(project_name)
 
 
+def register_workflow(name: Text, project_id: int, properties: Properties = None) -> WorkflowMeta:
+    return get_ai_flow_client().register_workflow(name=name, project_id=project_id, properties=properties)
+
+
+def get_workflow_by_name(project_name: Text, workflow_name: Text) -> Optional[WorkflowMeta]:
+    return get_ai_flow_client().get_workflow_by_name(project_name=project_name, workflow_name=workflow_name)
+
+
+def get_workflow_by_id(workflow_id: int) -> Optional[WorkflowMeta]:
+    return get_ai_flow_client().get_workflow_by_id(workflow_id)
+
+
+def list_workflows(project_name: Text, page_size: int, offset: int) -> Optional[List[WorkflowMeta]]:
```

Comment:
`list_workflows` interface should provide the function that lists workflows without paging.